Cargo.toml

```diff
@@ -47,6 +49,7 @@ lscolors = "0.7"
 globset = "0.4"
 anyhow = "1.0"
 dirs-next = "2.0"
+colored = "2.0.0"
```

Comment:
we already use the `ansi_term` dependency for colored output. It would be great if we could use it for this change as well.

Reply:
Sure, I've changed it to use `ansi_term` and removed `colored`

Reply:
I now remember why I added `colored`:

Using `ansi_term` I was unable to make the output colored **only** when the terminal supports it. This can for example be testet when piping to `cat`:

```
cargo run -- -h | \cat
```

At the current state, the last line containing the note for `--help` is still colored, which could lead to problems when the terminal does not support colors.

`colored` can somehow detect that and will disable coloring automatically.

Do you know a way to achieve the same with `ansi_term`?